### ISSUE TITLE
Tag all releases

### ISSUE THREAD

**Issue by sonicdoe** [react-native-community/react-native-blur#375]
[On npm](https://www.npmjs.com/package/@react-native-community/blur?activeTab=versions), there are five versions for `@react-native-community/blur`: 3.3.0, 3.3.1, 3.4.1, 3.5.0, and 3.6.0. However, [on GitHub](https://github.com/react-native-community/react-native-blur/tags), there are only Git tags for v3.3.0 and v3.3.1.

**Comment by m2mathew** [comment]
Agreed, it's always helpful to be able to reference release numbers and view specific changes that happen from version to version. Thanks for your work on this project, everyone!

**Comment by troZee** [comment]
Here you can find, what has been done in 3.6.0 

https://github.com/Kureev/react-native-blur/compare/d540dfbc58c1108f6dd41312538ee61c10e35455..887a5e029354a5d666d7c9afacc3e22631a36416
